test/test_methods.py

```diff
@@ -24,8 +24,11 @@
 import operator
 import os
 import random
+import time
 import warnings
 
+from Queue import Full, Empty
```

Comment:
Let's use `import Queue` instead, and then use `Queue.Full` and `Queue.Empty` below.